src/app.js

```diff
@@ -34,6 +52,16 @@ app.get('/', (req, res) => {
 })
 app.use("/shops", shopRoutes);
 
+
+// it calls user routes
+app.use("/user", userRoutes);
```

Comment:
the user management is handled gracefully by auth0, we have zero work on that regard :) I'll show you later man.

Reply:
> the user management is handled gracefully by auth0, we have zero work on that regard :) I'll show you later man.

Cool!